listenbrainz/webserver/templates/user/scraper.js

```diff
@@ -252,7 +254,8 @@ function reportPageAndGetNext(response, page) {
     }
     var struct = encodeScrobbles(response);
     submitQueue.push(struct);
-    if (struct['payload'][struct['payload'].length - 1]['listened_at'] <= latestImportTime) {
+    countReceived += struct['payload'].length;
+    if (struct['payload'][struct['payload'].length - 1]['listened_at'] < latestImportTime) {
```

Comment:
From commit message

```
Because we allow more than one listen with the
same timestamp now, we need to stop incremental
imports only if we encounter a listen older than the
latest imported one. Otherwise, some data can be lost,
for example if we have two pages of listens with the
same timestamp, the importer must not stop on the first
page
```